lsp-mode.el

```diff
@@ -7633,6 +7633,19 @@ When prefix UPDATE? is t force installation even if the server is present."
                             t))))
     (lsp--install-server-internal chosen-client t)))
 
+;;;###autoload
+(defun lsp-update-servers ()
+  "Update all installed servers."
+  (interactive)
+  (lsp--require-packages)
+  (cl-loop for client in
```

Comment:
consider using `mapc` here - it is a bit more coherent - e. g. it makes more sense to combine -filter with mapc because cl-loop has its own filtering facilities.

Reply:
done

Reply:
thanks, tested with `mapc`; it works